Submitted By: Jeremy Huntwork <jhuntwork@linuxfromscratch.org>
Date: 2005-03-23
Initial Package Version: 1.3.13
Upstream Status: Applied
Origin: Jeremy Huntwork
Description: Fixes Compile Error.

diff -Naur yaboot-1.3.13.orig/lib/strstr.c yaboot-1.3.13/lib/strstr.c
--- yaboot-1.3.13.orig/lib/strstr.c	2001-09-20 09:15:31.000000000 -0400
+++ yaboot-1.3.13/lib/strstr.c	2005-02-20 22:03:10.215796824 -0500
@@ -76,7 +76,9 @@
 	      a = *++haystack;
 	      if (a == '\0')
 		goto ret0;
-shloop:	    }
+shloop:	
+		a = a;
+	}
           while (a != b);
 
 jin:	  a = *++haystack;
